apis/v1alpha2/backendtlspolicy_types.go

```diff
@@ -0,0 +1,156 @@
+	// then a TLS connection must fail.
+	//
+	// References to a resource in a different namespace are
+	// invalid.
+	//
+	// A single CertRef to a Kubernetes ConfigMap kind has "Core"
+	// support.  Implementations MAY choose to support attaching
+	// multiple certificates to a backend, but this behavior is
+	// implementation-specific.  Also implementation-specific is
+	// a CertRef of other object kinds, e.g. Secret.
+	//
+	// Support: Core - An optional single reference to a Kubernetes
+	// ConfigMap.
+	//
+	// Support: Implementation-specific (No reference, more than one
+	// reference, or resource types other than ConfigMaps.
+	// Service mesh may ignore.)
+	//
+	// +kubebuilder:validation:MaxItems=8
+	// +optional
+	CertRefs []v1beta1.ConfigMapObjectReference `json:"certRefs,omitempty"`
+
+	// StandardCerts specifies whether system CA certificates may
+	// be used in the TLS handshake between the gateway and
+	// backend pod.
+	//
+	// If StandardCerts is unspecified or set to "", then CertRefs must
+	// be specified with at least one entry for a valid configuration.
+	// If StandardCerts is unspecified or set to "", then CertRefs must
+	// be specified.  Only one of CertRefs or StandardCerts may be
+	// specified, not both.
+	//
+	// StandardCerts must be set to "System" when CertRefs is unspecified.
+	//
+	// If StandardCerts is set to "System", then the system trusted
+	// certificates should be used. If there are none, or the
+	// implementation doesn't define system trusted certificates,
+	// then a TLS connection must fail.
+	//
+	// Support: Core - An optional value to specify whether to use
+	// system certificates or not.
+	//
+	// Support: Implementation-specific (In the absence of support
+	// for usable system certs, may be ignored. Service mesh may ignore.)
+	//
+	// +optional
+	StandardCerts *StandardCertType `json:"standardCerts,omitempty"`
+
+	// Hostname is the Server Name Indication that the Gateway uses to
+	// connect to the backend.  It represents the fully qualified domain
+	// name of a network host, as defined by RFC1123 - except that numeric
+	// IP addresses are not allowed. Each label of the FQDN must consist
+	// of lower case alphanumeric characters or '-', and must start and
+	// end with an alphanumeric character.  No other punctuation is allowed.
+	// Wildcard domain names are specifically disallowed.
+	//
+	// It specifies the hostname that may authenticate, and must be in the
+	// certificate served by the matching backend.
+	//
+	// Support: Core - A required value used by the Gateway to connect to
+	// the backend when a BackendTLSPolicy is specified.
+	Hostname v1beta1.PreciseHostname `json:"hostname"`
+}
+
+// StandardCertType is the type of CA certificate that will be used when
+// the TLS.certRefs is unspecified.
+// +kubebuilder:validation:Enum=System
+type StandardCertType string
+
+const (
+	StandardCertSystem StandardCertType = "System"
```

Comment:
```suggestion
	// Indicates that standard system CA certificates should be used.
	StandardCertSystem StandardCertType = "System"
```

Reply:
Regarding "standard", do we anticipate system certs other than standard, e.g. "optional" or "custom"?